### DISCUSSION TITLE
changing the URL setting

### DISCUSSION THREAD

**Discussion by edwardsmarkf** [katspaugh/wavesurfer.js#2876] [Q&A] [answered]
hi - in previous versions we could change the 'url' by merely doing   `waveSurfer.load(`

how do we change the urls now to select a different audio file?  

i am tempted to do a new `.create(` but i suspect there is a **FAR** better way.

EDIT:
maybe something like changing `wavesurfer.options.url= '...';` ??

**Comment by katspaugh** [comment, marked as answer]
Calling `load` multiple times should still work.

Destroying and creating it again is also OK, although it might be slightly slower than just loading a new URL.

Btw, other options can be set via `setOptions`.
